xapian-applications/omega/omega.cc

```diff
@@ -361,8 +366,11 @@ try {
     for (auto i = begin; i != end; ++i) {
 	const string & v = i->second;
 	if (!v.empty()) {
-	    Xapian::valueno slot = atoi(i->first.c_str() +
-					CONST_STRLEN("START."));
+	    Xapian::valueno slot;
+	    if (!parse_unsigned(i->first.c_str() +
+		CONST_STRLEN("START."), slot)) {
```

Comment:
I just noticed this (and a similar case below) are misaligned - the `C` of `CONST_STRLEN` should line up with the `i` of `i->first.c_str()`, since they're both parameters of `parse_unsigned`.  As currently indented, this looks as if `CONST_STRLEN(...)` is being added to the result of `!parse_unsigned(...)`.

I should have spotted that before, so in the interests of not dragging this review out further I'll fix this up as I merge (I'll need to adjust as I merge anyway to change the git commit hashes in `bootstrap` so they refer to a commit which actually exists in the history after squashing all the fixup commits).